content/en/blog/_posts/2023-03-08-nodeport-dynamic-and-static-allocation.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: blog
+title: "Kubernetes 1.27: Avoid Collisions Assigning ports to Nodeport Services"
+date: 2023-03-08
+slug: nodeport-dynamic-and-static-allocation
+---
+
+**Author:** xuzhenglun
+
+In Kubernetes, the Service can be used to provide a unified traffic entry point for applications running on a set of Pods. Clients can use the virtual IP address provided by the Service for access, and Kubernetes provides load balancing for traffic accessing different back-end Pods, but ClusterIP-type Service is limited to providing access to nodes within the cluster, and traffic from outside the cluster cannot be routed. One way to solve this problem is to use NodePort-type Service, which maps a VIP's port to a specific port of all nodes in the cluster, thus redirecting traffic from the outside to the inside of the cluster.
```

Comment:
Wrap long lines like this properly.
It will help reviewers to provide feedbacks;
and it will benefit downstream localization teams for translation and change tracking.

Reply:
It's done